graphistry/feature_utils.py

```diff
@@ -2193,8 +2315,7 @@ def scale(
                     'Please run g.featurize(kind="edges", *args, **kwargs) '
                     'first before scaling matrices and targets is possible.'
                 )
-
-        return X, y, scaling_pipeline, scaling_pipeline_target
+        return X, y
```

Comment:
this will affect `graphistry/tests/test_feature_utils.py::TestFeatureProcessors::test_process_node_dataframes_min_words` as the its expecting 4 outputs